python/uv/Dockerfile

```diff
@@ -31,11 +34,14 @@ RUN --mount=type=cache,target=/root/.cache/uv \
 # Final image, minimal runtime
 FROM base
 
+# Set working directory
+WORKDIR /app
+
 # Copy full app + .venv from builder
 COPY --from=builder /app /app
 
 # Activate virtualenv
 ENV PATH="/app/.venv/bin:$PATH"
 
 # Launch app (adjust if your module path changes)
-CMD ["uvicorn", "uv_docker_example:app", "--host", "0.0.0.0", "--port", "$PORT"]
```

Comment:
I think the port should still be dynamically assigned, it helps ensure Suga can deploy multiple services to the same host. What error is this causing, maybe there is another fix